script/tool/lib/src/create_all_plugins_app_command.dart

```diff
@@ -259,4 +268,75 @@ dev_dependencies:${_pubspecMapString(pubspec.devDependencies)}
 
     return buffer.toString();
   }
+
+  Future<int> _genNativeBuildFiles() async {
+    // Only run on macOS.
+    // Other platforms don't need generation of additional files.
```

Comment:
Could you extract the script/tool/ change to a separate PR? It needs some review, and also its own test changes, and since it's largely orthogonal to the rest of the PR I think it would be easier to iterate and land it separately. Even though this PR is what makes this change necessary, it's fine to pre-land the tool change.

(Extracting this part of the PR will also make the presubmit tests for this PR run much faster, since having tool changes in this PR causes tests for *all* plugins to run.)

Reply:
Just to be sure: these tool changes should be a (third) PR, next to this one and a separate PR for the integration test fix?

Reply:
Extracted the tool change to https://github.com/flutter/plugins/pull/6605.